.agents/skills/chrome-devtools/SKILL.md

```
@@ -0,0 +1,40 @@
+name: chrome-devtools
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)